dub.json

```diff
@@ -12,10 +12,10 @@
     "StdLoggerDisableWarning"
   ],
   "dependencies" : {
-    "libdparse" : "~>0.7.2-alpha.5",
-    "dsymbol" : "~>0.2.6",
+    "libdparse" : "~>0.8.0-alpha.1",
```

Comment:
![screenshot_20180129_124230](https://user-images.githubusercontent.com/16154339/35508799-fe08d200-04f1-11e8-9f35-2835588fb8f6.png)

[`Root package dscanner reference libdparse >=0.8.0-alpha.1 <0.9.0-0 cannot be satisfied.`](https://travis-ci.org/dlang-community/D-Scanner/jobs/334663763#L491)

Reply:
Must be a network error. DUB semver.d verifies well `assertLess("0.8.0-alpha.1", "0.9.0");`

Reply:
Even with "*" as version DUB fails (with message `Root package dscanner reference libdparse >=0.0.0 cannot be satisfied.`). I don't know what to do except reporting a bug at DUB's bugtracker.

Reply:
DUB -vverbose:

```
Configurations used for dependency resolution:
  dsymbol (0, required): [0.3.0-beta.1]
  emsi_containers (1, required): [0.6.0]
  inifiled (3, required): [1.0.2]
  libddoc (4, required): [0.2.0]
  libdparse (2, required): [invalid]
Package dsymbol contains invalid dependency libdparse (no version candidates) (ci=0)
Package libddoc contains invalid dependency libdparse (no version candidates) (ci=4)
Root package dscanner reference libdparse >=0.8.0-alpha.1 <0.9.0-0 cannot be satisfied.
Full exception: object.Exception@source/dub/dependencyresolver.d(158): Root package dscanner reference libdparse >=0.8.0-alpha.1 <0.9.0-0 cannot be satisfied.
```